common/src/main/scala/explore/undo/Action.scala

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2016-2021 Association of Universities for Research in Astronomy, Inc. (AURA)
```

Comment:
It looks like most of the changes to the undo stuff is just removing the parameterization on `F`?

Reply:
Yes. Instead of using tagless final for the effects (which involved a sync and an async in `UndoContext`) we now just use the ones defined by default in the `scalajs-react` bundle (which in our case is `CallbackTo` and `IO`).